### PR TITLE
Yarn Migration

### LINKED ISSUE #1124: Switch To Yarn
### Please describe the problem (or idea)
Gargantuan packages like puppeteer take very long to install. Installing them repeatedly becomes tedious. I face this a lot when pushing to beta. 
The caching feature of yarn can solve this issue.

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by HarshKhandeparkar**
Fixes #1124  (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
yarn warnings
```
warning istanbul@0.4.5: This module is no longer maintained, try this instead:
  npm i nyc
Visit https://istanbul.js.org/integrations for other alternatives.
warning grunt-contrib-uglify-es@3.3.0: deprecated
warning tape-run > browser-run > ecstatic@4.1.2: https://github.com/jfhbrook/node-ecstatic/issues/259
warning tape-run > browser-run > electron-stream > ecstatic@4.1.2: https://github.com/jfhbrook/node-ecstatic/issues/259
warning tape-run > browser-run > html-inject-script > trumpet > html-tokenize > through2 > xtend > object-keys@0.4.0: 
[2/4] Fetching packages...
error node-png@0.4.3: The engine "node" is incompatible with this module. Expected version "0.8.x". Got "10.16.0"
```

We have to improve upon these

File: README.md [hunk 1]
```diff
@@ -84,16 +90,16 @@ Add `image-sequencer` to your list of dependencies and run `npm install`
 
 Globally install Image Sequencer:
 
-```
-$ npm install image-sequencer -g
+```sh
+npm install image-sequencer -g
```
Comment (alaxalves) [inline README.md]: Shouldn't we change this to `yarn global add image-sequencer`??

**Comment by HarshKhandeparkar** [comment]
These issues are interesting. We have to somehow track this node-png module. It asks for node v0.8.x it is extremely outdated. Also we'll have to fix the warnings.

cc @jywarren @tech4GT @publiclab/is-reviewers @rexagod

File: README.md (hunk 1 above)
Comment (HarshKhandeparkar) [reply README.md]: I will change that later. Currently I would like to fix the errors and warnings. The deps have to be updates. Thanks!

**Comment by HarshKhandeparkar** [comment]
@jywarren I can see that the package node-png works in node 8. It was also confirmed that gpu.js works in node 8(I personally also have used it in node 8). The problem are the warnings, the errors can be forced but we have to update our other dependencies based on the warnings.

Yarn is a really good package manager and it will give warnings in the future too. This is a very important step.

cc @publiclab/is-reviewers @Divy123 @aashna27 @alaxalves

**Comment by jywarren** [comment]
Hi, all, I'd love to hear from others on this @publiclab/is-reviewers -- thanks @HarshKhandeparkar !!!

**Comment by HarshKhandeparkar** [comment]
Merging this PR isn't as important as fixing those errors is. @jywarren

**Comment by jywarren** [comment]
Sorry, which errors? Can you catch me up by linking? I'm behind on a huge backlog of issues right now, thanks for the help!

**Comment by HarshKhandeparkar** [comment]
https://github.com/publiclab/image-sequencer/pull/1141#issuecomment-507351374. Sorry

**Comment by vaibhavmatta** [comment]
Have u tried yarn --ignore-engine ?

**Comment by HarshKhandeparkar** [comment]
That is what is making it work.

**Comment by vaibhavmatta** [comment]
I think this is the only option left now.! 
But anyhow, updated Node version will definitely make issues if a lower version is required. I faced same problems during my internship at a startup. 
Let us try a bit more and find out a way to overcome this.

**Comment by vaibhavmatta** [comment]
Try changing version of up-path in Yarn.lock file..
@HarshKhandeparkar DM me on Gitter, I think I found the alternative.

**Comment by HarshKhandeparkar** [comment]
Sure. Just that I don't know what to do. I really appreciate your help.
Thanks !

**Comment by vaibhavmatta** [comment]
I've sent you the solution on Gitter..! Try applying it. 
Ty

**Comment by HarshKhandeparkar** [comment]
Meh

**Pull request closed without merging**